### PR TITLE
Updated README.md to add my name

### PR DESCRIPTION
**Pull request by michellejyao**
added my name to readme

### CODE REVIEW

File: README.md
Comment (pijner) [inline README.md]: Can you remove these additional added lines so the diff is clean?

**Review by pijner — CHANGES_REQUESTED** [review]
Can you remove these additional added lines so the diff is clean?

File: README.md
Comment (michellejyao) [reply README.md]: i deleted the newlines on my end. when i saved files on vsc the prettier extension would add newlines automatically. i dont think i can send a second pull request with the deleted newlines

Comment (pijner) [reply README.md]: Kk. Not a big deal.
If this becomes a problem when working with code files, you can probably use a `.prettierignore` file or disable format on save in vac for this project

**Review by pijner — APPROVED** [review]
Good to merge once merge conflict is resolved

**Pull request merged**
